packages/services/rate-limit/package.json

```diff
@@ -15,10 +15,13 @@
     "@sentry/tracing": "7.21.1",
     "@trpc/client": "10.4.3",
     "@trpc/server": "10.4.3",
+    "@valu/trpc-fetch-api-adapter": "^0.0.3",
     "@whatwg-node/fetch": "0.5.3",
+    "@whatwg-node/server": "^0.4.17",
     "date-fns": "2.29.3",
     "dotenv": "10.0.0",
     "got": "12.5.3",
+    "itty-router": "2.6.6",
```

Comment:
@ardatan you mentioned that `itty-router` have some issues recently? should we use it here?

Reply:
curious about that 👀